prometheus/metrics.go

```diff
@@ -96,10 +103,22 @@ func getServiceMetrics(api v1.API, namespace string, servicename string, version
 		versionLabelOut = fmt.Sprintf(",source_version=\"%s\"", version)
 	}
 
-	labelsIn := fmt.Sprintf("{destination_service=\"%s.%s.%s\"%s}", servicename, namespace, clustername, versionLabelIn)
-	labelsOut := fmt.Sprintf("{source_service=\"%s.%s.%s\"%s}", servicename, namespace, clustername, versionLabelOut)
-	labelsErrorIn := fmt.Sprintf("{destination_service=\"%s.%s.%s\",response_code=~\"[5|4].*\"%s}", servicename, namespace, clustername, versionLabelIn)
-	labelsErrorOut := fmt.Sprintf("{source_service=\"%s.%s.%s\",response_code=~\"[5|4].*\"%s}", servicename, namespace, clustername, versionLabelOut)
+	var labelsIn, labelsOut, labelsErrorIn, labelsErrorOut string
+	if op == "" || op == EQUALS {
+		labelsIn = fmt.Sprintf("{destination_service=\"%s.%s.%s\"%s}", servicename, namespace, clustername, versionLabelIn)
+		labelsOut = fmt.Sprintf("{source_service=\"%s.%s.%s\"%s}", servicename, namespace, clustername, versionLabelOut)
+		labelsErrorIn = fmt.Sprintf("{destination_service=\"%s.%s.%s\",response_code=~\"[5|4|3].*\"%s}", servicename, namespace, clustername, versionLabelIn)
```

Comment:
3XX response code are not errors

Reply:
uh oh.... now that you mention it :)
all the error % and error rate calcs we do in the UI are simply sums of the 3xx, 4xx, and 5xx codes. We should probably change all of that to only consider 4xx and 5xx as errors.

Reply:
True, but it's also not success. We need to know how to treat these consistently because right now the graph generator lumps them in with 4xx and 5xx when calculating percent error.  I don't care either way, just want to be consistent so that is why I updated it here to match the generator.

Reply:
see PR https://github.com/kiali/swsui/pull/162

The UI no longer will consider 3xx errors (they really aren't errors in the sense we care about - we don't want to our "%errors" to include things like being told to redirect).

Reply:
I wonder if 3xx should just be ignored? Ie. sum of requests would be successes (2xx) + errors (4xx, 5xx), and that's it?